pkg/commands/root.go

```diff
@@ -89,6 +106,26 @@ func (e *Executor) persistentPostRun(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func (e *Executor) needVersionOption() bool {
+	return e.buildInfo.Date != ""
+}
+
+func initRootFlagSet(fs *pflag.FlagSet, cfg *config.Config, needVersionOption bool) {
+	fs.BoolVarP(&cfg.Run.IsVerbose, "verbose", "v", false, wh("Verbose output"))
+	fs.StringVar(&cfg.Output.Color, "color", "auto", wh("Use color when printing; can be 'always', 'auto', or 'never'"))
+
+	fs.StringVar(&cfg.Run.CPUProfilePath, "cpu-profile-path", "", wh("Path to CPU profile output file"))
+	fs.StringVar(&cfg.Run.MemProfilePath, "mem-profile-path", "", wh("Path to memory profile output file"))
+	fs.StringVar(&cfg.Run.TracePath, "trace-path", "", wh("Path to trace output file"))
+
+	fs.IntVarP(&cfg.Run.Concurrency, "concurrency", "j", getDefaultConcurrency(),
+		wh("Number of CPUs to use (Default: number of logical CPUs)"))
+
+	if needVersionOption {
```

Comment:
question: why do not print version always? and if no build just print `(devel)` or something

Reply:
UPD: condition will be removed